grunt/helpers.js

```diff
@@ -245,6 +279,11 @@ module.exports = function(grunt) {
         return configValue || grunt.config('includedRegExp', generateIncludedRegExp());
     };
 
+    exports.getNestedIncludedRegExp = function() {
+        var configValue = grunt.config('nestedIncludedRegExp');
+        return configValue || grunt.config('nestedIncludedRegExp', generateNestedIncludedRegExp());
+    };
+
     exports.getExcludedRegExp = function() {
         var configValue = grunt.config('excludedRegExp');
         return configValue || grunt.config('excludedRegExp', generateExcludedRegExp());
```

Comment:
make the lower functions private as @danielstorey suggests

Reply:
Done